_includes/v3.10/release-notes/v3.10.0-release-notes.md

```diff
@@ -0,0 +1,59 @@
+ - Add semaphore config. [node #324](https://github.com/projectcalico/node/pull/324) (@neiljerram)
+ - The default IP pool block size is now configurable. Use the `CALICO_IPV4POOL_BLOCK_SIZE` and `CALICO_IPV6POOL_BLOCK_SIZE` environment variables to configure the default pool block size. [node #346](https://github.com/projectcalico/node/pull/346) (@halfcrazy)
+ - The `CALICO_IPV4POOL_NODE_SELECTOR` and `CALICO_IPV6POOL_NODE_SELECTOR` environment variables were added for configuring default pool node selectors. [node #346](https://github.com/projectcalico/node/pull/346) (@tmjd)
+ - Add calicoctl `patch` command similar to `kubectl patch`. [calicoctl #2054](https://github.com/projectcalico/calicoctl/pull/2054) (@syscll)
+ - Update calicoctl alpine image to 3.10. [calicoctl #2061](https://github.com/projectcalico/calicoctl/pull/2061) (@chenrui333)
+ - Re-enable golangci-lint linters and fix lint warnings. [felix #2130](https://github.com/projectcalico/felix/pull/2130) (@rafaelvanoni)
+ - Add support to watch stderr for FV containers. [felix #2137](https://github.com/projectcalico/felix/pull/2137)(@song-jiang)
+ - Reduce severity of BPF log levels. [felix #2144](https://github.com/projectcalico/felix/pull/2144) (@tmjd)
+ - Fix felix log level. [felix #2147](https://github.com/projectcalico/felix/pull/2147) (@neiljerram)
+ - Clean up code. [felix #2150](https://github.com/projectcalico/felix/pull/2150) (@beautytiger)
+ - Run typha as non-root user. [typha #312](https://github.com/projectcalico/typha/pull/312)
+ - Improve log for check command failing. [confd #271](https://github.com/projectcalico/confd/pull/271) (@neiljerram)
+ - Move some testutils functions to utils_linux.go. [cni-plugin #787](https://github.com/projectcalico/cni-plugin/pull/787) (@song-jiang)
+ - Use 'git diff' instead of 'git diff-index' in Makefile's git-commit target. [cni-plugin #788](https://github.com/projectcalico/cni-plugin/pull/788) (@rafaelvanoni)
+ - Add assertion that CI builds do not result in dirty state. [cni-plugin #790](https://github.com/projectcalico/cni-plugin/pull/790) (@caseydavenport)
+ - Handle removing node for flannel migration controller. [kube-controllers #404](https://github.com/projectcalico/kube-controllers/pull/404) (@song-jiang)
+ - Add flannel migration FV. [kube-controllers #406](https://github.com/projectcalico/kube-controllers/pull/406) (@song-jiang)
+ - Use a real logrus version. [libcalico-go #1124](https://github.com/projectcalico/libcalico-go/pull/1124) (@rafaelvanoni)
+ - Ignore Windows reserved addresses. [libcalico-go #1129](https://github.com/projectcalico/libcalico-go/pull/1129) (@caseydavenport)
+ - Use Kubernetes 1.16 in tests. [libcalico-go #1131](https://github.com/projectcalico/libcalico-go/pull/1131) (@caseydavenport)
+ - Fix code warnings. [libcalico-go #1132](https://github.com/projectcalico/libcalico-go/pull/1132) (@beautytiger)
```

Comment:
Might want to combine this one or move it closer to the other code cleanup one from beautytiger?